dts/arm/nxp/nxp_k8x.dtsi

```diff
@@ -432,17 +432,16 @@
 			dma-requests = <64>;
 			nxp,mem2mem;
 			reg = <0x40008000 0x1000>,
-				<0x40021000 0x1000>;
+			      <0x40021000 0x1000>;
 			interrupts = <0 0>, <1 0>, <2 0>, <3 0>,
-				<4 0>, <5 0>, <6 0>, <7 0>,
-				<8 0>, <9 0>, <10 0>, <11 0>,
-				<12 0>, <13 0>, <14 0>, <15 0>,
-				<16 0>;
+				     <4 0>, <5 0>, <6 0>, <7 0>,
+				     <8 0>, <9 0>, <10 0>, <11 0>,
+				     <12 0>, <13 0>, <14 0>, <15 0>,
+				     <16 0>;
 			clocks = <&sim KINETIS_SIM_DMA_CLK 0x1040 0x00000002>,
-				<&sim KINETIS_SIM_DMAMUX_CLK 0x103C 0x00000002>;
+				 <&sim KINETIS_SIM_DMAMUX_CLK 0x103C 0x00000002>;
```

Comment:
i am not understanding these formatting changes ? it doesn't seem to align with anything and just adds spaces for some reason I dont understand? unless github is showing it wrong?

Reply:
Will open this in an editor tonight and share screenshots for easier understanding

Reply:
I wonder if there is some way to make github show this type of thing properly